EosGround/config/create_database.sql

```diff
@@ -0,0 +1,13 @@
+-- Database: eos_db
+
+-- DROP DATABASE IF EXISTS eos_db;
+
+CREATE DATABASE eos_db
+    WITH
+    OWNER = postgres
+    ENCODING = 'UTF8'
+    LC_COLLATE = 'en_US.UTF-8'
+    LC_CTYPE = 'en_US.UTF-8'
+    TABLESPACE = pg_default
+    CONNECTION LIMIT = -1
+    IS_TEMPLATE = False;
```

Comment:
This will fail on locale errors unless you include the following `TEMPLATE template0`
In other words, add that line to this query and I'll approve it